### PR TITLE
[WIP] Start storing pipeline data to db

### PR DESCRIPTION
**Pull request by rtnpro**
This implements (or will) saving pipeline state/data to DB so that we can churn various useful metrics from it, and also make our workers fault tolerant.

### CODE REVIEW

**Comment by navidshaikh** [comment]
I'll work on completing this feature.

**Comment by navidshaikh** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
After customizing ``allowed_hosts`` in ``[all:vars]`` in ansible hosts file and deploying tags ``db``, ``django``, ``application`` you can run

``/opt/cccp-service/manage.py runserver 0.0.0.0:8081`` in jenkins master node to start django server for admin panel. You can then access the admin panel at ``<an allowed_host>:8081``.

Creating superuser:

``/opt/cccp-service/manage.py createsuperuser``

TODO:

- [ ] automate creating super user
- [ ] link different admin panels into a consolidated view
- [ ] defining policies around db interaction

**Comment by navidshaikh** [comment]
#dotests-debug

File: provisions/roles/repo_tracking/tasks/main.yml [hunk 1]
```diff
@@ -4,8 +4,13 @@
   sudo: yes
   tags: repo_tracking
 
+- name: Install pip
+  yum: name=python-pip state=installed
+  sudo: yes
+  tags: repo_tracking
+
```
Comment (navidshaikh) [inline provisions/roles/repo_tracking/tasks/main.yml]: We are anyway installing these pip, django, etc for 3 nodes including jenkins_master with
https://github.com/CentOS/container-pipeline-service/pull/302/commits/0a60e279ac26f9bc6e10df5a22b099eaef118b56

We dont need this part of code in this commit.

**Comment by navidshaikh** [comment]
#dotests-debug

**Comment by navidshaikh** [comment]
#dotests-debug

**Comment by navidshaikh** [comment]
The deployment is failing for me for step
```
TASK [db : Pull CentOS postgres container] *************************************
task path: /home/centos/container-pipeline-service/provisions/roles/db/tasks/main.yml:15
fatal: [dev-33-27.lon1.centos.org]: FAILED! => {"changed": false, "failed": true, "msg": "Failed to import docker-py - No module named docker. Try `pip install docker-py`"}
```

Looking into it.

**Comment by navidshaikh** [comment]
[test] GH-MM comments integration

**Comment by navidshaikh** [comment]
Closing this PR in the favor of https://github.com/CentOS/container-pipeline-service/pull/356 , all the code is rebased with that PR and the feature will be worked upon over there.

**Pull request closed without merging**